docs/source/data/data_catalog.md

```diff
@@ -614,6 +614,6 @@ Finally we can save the processed data in Parquet format.
 io.save("ranked", ranked)
 ```
 
-```eval_rst
-.. attention::  Saving ``None`` to a dataset is not allowed!
+```{warning}
```

Comment:
From documentation contributor guidelines: "Prefer to use `note`, `warning` and `important` only, rather than a number of different colours/types of callout." I tried to color match to the appropriate level.